app/ui/browser-blueprint/views/overview/overview-cards.jsx

```diff
@@ -32,7 +33,12 @@ const OverviewCards = function(props) {
       {props.pages.map((page, pageIndex) => (
         <OverviewCard key={`page-${page.id}`}
           page={page}
-          pageIndex={pageIndex}
+          width={UIConstants.CARD_WIDTH}
```

Comment:
Is there a reason to prefer passing in the constants from OverviewCards to OverviewCard instead of requiring the UIConstants in OverviewCard? I think we want to avoid threading properties through from parent to child components when possible.

If we were going to be reusing overview-card independently of overview-cards (like if we had multiple different overview-cards components for different themes), then I could see passing them in as props being an advantage. But from a quick look at the current component file structure, I didn't see an indication that we're doing this.